docs/xmlprofiles.rst

```diff
@@ -817,6 +861,15 @@ This section of the :class:`Participant's rtps` configuration allows defining bu
 |                            | is set to :class:`STATIC`             |                         |                      |
 +----------------------------+---------------------------------------+-------------------------+----------------------+
 
+.. _Participantfiltering:
+
+**ignoreParticipantFlags**
+
+* **FILTER_DIFFERENT_HOST** all metadata from another host would be discarded.
+* **FILTER_DIFFERENT_PROCESS** all metadata from another process on the same host would be discarded.
+* **FILTER_SAME_PROCESS** all metadata from our own process would be discarded.
+* **FILTER_DIFFERENT_PROCESS | FILTER_SAME_PROCESS** all metadata from our own host would be discarded.
```

Comment:
I think this can be placed in the discovery config table in the same way that <EDP> tag possibilities where broken down